spring-content-commons/src/main/java/org/springframework/content/commons/io/MedializedResource.java

```diff
@@ -0,0 +1,90 @@
+package org.springframework.content.commons.io;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URL;
+
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.core.io.Resource;
+
+public class MedializedResource implements MediaResource {
```

Comment:
Not sure I like the name.  How about just MediaResourceImpl, or DefaultMediaResource?

Reply:
I Agree,  "DefaultMediaResource" OK?

Reply:
I like DefaultMediaResource